client/js/socket-events/msg.js

```diff
@@ -35,7 +35,8 @@ socket.on("msg", function(data) {
 		.trigger("msg", [
 			target,
 			data
-		]);
+		])
+		.trigger("keepToBottom");
```

Comment:
Because `keepToBottom` has logic inside to only "keep to bottom" if it's already at the bottom. Thats's the point of the plugin.